.travis.yml

```diff
@@ -7,4 +7,4 @@ before_install:
 
 script:
     - python3 -m unittest discover -s metatests
```

Comment:
We don't really see much info about successful tests in the travis output.  Perhaps add a "verbose" option for the unit tests?

Reply:
Fixed in #9